qa/acceptance-tests/src/test/java/io/camunda/it/client/ProcessInstanceSearchTest.java

```diff
@@ -1418,16 +1418,4 @@ void shouldReturnOnlyMatchingTopLevelWhenOrClauseErrorMessageConflicts() {
 
     assertThat(result.items()).isEmpty();
   }
-
-  @Test
-  void shouldReturnEmptyResultForIncorrectIncidentErrorHashCode() {
```

Comment:
❓ Why did we delete this test? Is this unrelated?

Reply:
this probably happened when I extracted the other test cases from this file into a dedicated test.
🙇 I should have been more careful and double check the commit (it's been a long day already)